src/testing/workspace-fixtures.ts

```diff
@@ -0,0 +1,52 @@
+import { defaultLocation } from 'src/analysis/utils/runtime-utils';
+import { locationTypes } from 'src/components/region-common';
+import { AzureWorkspace, GoogleWorkspace } from 'src/libs/workspace-utils';
+
+export const defaultAzureWorkspace: AzureWorkspace = {
+  workspace: {
+    authorizationDomain: [],
+    cloudPlatform: 'Azure',
+    name: 'test-azure-ws-name',
+    namespace: 'test-azure-ws-namespace',
+    workspaceId: 'fafbb550-62eb-4135-8b82-3ce4d53446af',
+    createdDate: '2023-02-15T19:17:15.711Z',
+    createdBy: 'justin@gmail.com',
+  },
+  azureContext: {
+    managedResourceGroupId: 'test-mrg',
+    subscriptionId: 'test-sub-id',
+    tenantId: 'test-tenant-id',
+  },
+  accessLevel: 'OWNER',
+  canShare: true,
+  canCompute: true,
+};
+
+export const defaultAzureStorageOptions = {
```

Comment:
Happy to put something legit in here, but I actively avoided reusing the one in `ContextBar` because it's the only one that did so.  Most of the other tests are focused on GCP and therefore leaving `azureStorageOptions` undefined.

One thought: to make this explicit, I could provide another export called `emptyAzureStorageOptions`, and then refer to that throughout the tests that are focused on GCP.  But before I take that on I want to make sure it's reasonable.  WDYT?

Reply:
Done.  Most of the tests were happy to use the more intelligent defaults.  A couple tests explicitly wanted `emptyAzureStorageOptions`.